engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -23,7 +23,7 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
+  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson] = None, workflowOptions: Option[WdlJson] = None): Props = {
```

Comment:
@cjllanwarne @geoffjentry @Horneth 
1. VAs-per-request appeals more to me because of the reason already mentioned by @Horneth .

2.) I think passing the arguments with the actor (and not in the message) makes more sense in this case where we are creating VAs per request, as it ties that VA to just one workflow (request). Passing it in message, you can potentially send messages with different workflows to this actor (which breaks the very tenet that we want single VAs per request). On a tangential note, this same logic applied to instantiation of a Backend in PBE where we used a single TaskDescriptor for instantiating a backend, which not only ties a backend to the philosophy of task-at-a-time model, but avoids passing a TaskDesc each time to prepare(), execute(), cleanup() etc. where it is difficult to guarantee that you are in fact passing the same instance of TaskDesc to each of those methods.